setup.py

```diff
@@ -23,31 +23,29 @@
 
 
 def get_requirements():
-    """Parse all packages mentioned in the 'requirements.txt' file."""
-    with open('requirements.txt') as fd:
+    """Parse dependencies from 'requirements.in' file."""
+    with open('requirements.in') as fd:
         lines = fd.read().splitlines()
-        reqs, dep_links = [], []
+        requires = []
         for line in lines:
-            if line.startswith('git+'):
-                dep_links.append(line)
-            else:
-                reqs.append(line)
-        return reqs, dep_links
+            requires.append(line)
```

Comment:
This loop is Just to append stuff from `lines` to `requires`..
Why we need this Loop..? why can't you dump `lines` data in `requires` if really needed..?

Reply:
Yes, the loop is not required. I had taken it from Arun's PR (https://github.com/fabric8-analytics/fabric8-analytics-utils/pull/58/files) as he suggested in one of my PR. 
That point of time I did not notice this, I have removed loop as you suggested.